scrabble_challenge/scrabble_challenge.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+CHARACTERS_SCORE = {"a": 1, "c": 3, "b": 3, "e": 1, "d": 2, "g": 2,
+                    "f": 4, "i": 1, "h": 4, "k": 5, "j": 8, "m": 3,
+                    "l": 1, "o": 1, "n": 1, "q": 10, "p": 3, "s": 1,
+                    "r": 1, "u": 1, "t": 1, "w": 4, "v": 4, "y": 4,
+                    "x": 8, "z": 10}
+
+
+def get_scrabble_score(valid_word):
+    """
+    Calculates the scrabble score of valid words and returns it.
+    """
+    scrabble_score = 0
+    for character in valid_word:
+        scrabble_score += CHARACTERS_SCORE[character]
+    return scrabble_score
+
+
+def is_word_from_rack(word, rack):
+    """
+    This function check the word that if it is made of the letters that are subset of rack letters.
+    """
+    rack = str(rack)
+    rack = rack.lower()
+    for character in word:
+        if character not in rack:
+            return False
+        else:
+            rack = rack.replace(character, "", 1)
+    return True
+
+
+def display_score(valid_words_list):
+    """
+    This function takes the list of valid words and then get their score and dispalys final result.
+    """
+    if valid_words_list != []:

```

Comment:
Try not to use words like list and dictionary in variable names. 
`valid_words_scores` instead of `valid_words_dictionary`. Also look for other variable names which can be improved.